samples/deployment/size/m-cluster/amster.yaml

```diff
@@ -0,0 +1,22 @@
+image:
+  repository: forgerock-docker-public.bintray.io/forgerock/amster
+  tag: 6.0.0
+  pullPolicy: Always
+
+# Defines default global values. These can get overridden with -f custom.yaml
+global:
+  domain: .frk8s.net
+  git:
+    # git repo to clone. The value below is a public git repo that does not require authentication.
```

Comment:
Just verifying that the plan is to make cloud-deployment-config public. It's currently private and I hadn't heard whether we will make it public or not. 

Also, when we get to a release, you're probably going to need to change branch: master to something like branch: release/6.5.0

This appears in other places in the PR, so pls change globally if you decide you want to change it here.